test/configs/dancebox.yml

```diff
@@ -12,3 +12,5 @@ import-storage:
                     free: "100000000000000000000000"
     Sudo:
         Key: "5GrwvaEF5zXb26Fz9rcQpDWS57CtERHpNehXCPcNoHGKutQY"
+    PolkadotXcm:
```

Comment:
Where do we use these constants?

Reply:
these are used so that we reset the query counter. it's something very much xcm specific:

- the xcm migration mostly migrate queries and locked assets. the latter are a bit difficult to create, so I chose to go with the first.
- in order to create a xcm query, you need a valid chain to which you can talk to. in the case of parachains, this is usually the relay chain. However some of them **might already have a query to the relay chain**, hence the reason for us to reset the counter to 0, so that we start fresh.
- in the case of the solo chain, we simply create a fake query to a fake para-id. we cannot do this in the parachain context as it is a more complex and convoluted thing to mock